lesson11/main.js

```diff
@@ -1,5 +1,26 @@
 const ul = document.getElementById("js-ul"); 
 
+// const url = "";
+// const url = "https://mocki.io/v1/55dc6233-a8fe-44ca-8906-3de313545ce8";
+const url = "https://mocki.io/v1/1c058349-634e-462a-ad37-14f135e59b99";
+const options = {
+  method: "GET"
+};
+
+function renderStatus(error){
+	const p = document.createElement("p"); 
+  p.id = "render-status";
+  p.textContent = `${error.status}:${error.statusText}`;
+  document.body.appendChild(p);
+}

```

Comment:
ここ、変数名はレスポンスをそのまま使用したほうがいいと思いました。
エラーというよりは、サーバーからのレスポンスだと思うので。

```suggestion
function renderStatus(response){
	const p = document.createElement("p"); 
  p.id = "render-status";
  p.textContent = `${response.status}:${response.statusText}`;
  document.body.appendChild(p);
}

```

Reply:
Thank you chihiro-san:)
a8900373cfa23d3bedcacd58640379f5b03c62f5